apps/web/components/dev/seeding-tab.tsx

```
@@ -0,0 +1,186 @@
+            n.pinned ? 'pinned: true' : null,
+            n.favorite ? 'favorite: true' : null,
+            n.folder ? `folder: '${escape(n.folder)}'` : null
+        ].filter(Boolean).join(', ')
+
+        return `    { ${props} },`
+    }).join('\n')}
+]
+`
+}
+
+function escape(str: string) {
```

Biome (2.1.2)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)